specification.md

```diff
@@ -145,10 +163,47 @@ configuration.
 See sections under the [Operation](#operation) section for the details of the
 message sequences.
 
+The WebSocket transport is typically used when it is necessary to have instant
+communication ability from the Server to the Agent without waiting for the Agent
+to poll the Server like it is done when using the HTTP transport (see below).
+
+## Plain HTTP Transport
+
+The second supported transport for OpAMP protocol is plain HTTP connection. The
+Agent is an HTTP client and the Server is an HTTP client server. The Agent makes
+POST requests to the Server. The body of the POST request and response is a
+[binary serialized Protobuf](https://developers.google.com/protocol-buffers/docs/encoding)
+message. The Agent sends AgentToServer Protobuf messages in the request body and
+the Server sends ServerToAgent Protobuf messages in the response body.
+
+OpAMP over HTTP is a synchronous, half-duplex message exchange protocol. The
+Agent initiates an HTTP request when it has an AgentToServer message to deliver.
+The Server responds to each HTTP request with a ServerToAgent message it wants
+to deliver to the Agent. If the Agent has nothing to deliver to the Server the
+Agent MUST periodically poll the Server by sending an AgentToServer message
+where only [instance_uid](#instance_uid) field is set. This gives the Server an
+opportunity to send back in the response any messages that the Server wants to
+deliver to the Agent (such as for example a new remote configuration).
```

Comment:
I think this is the right approach. Polling also acts as a heartbeat from the agent